stylelint.config.js

```diff
@@ -32,6 +35,9 @@ module.exports = {
     'at-rule-disallowed-list': ['debug'],
     'scss/at-rule-no-unknown': true,
 
+    // Disable unwanted scss rules
+    'scss/comment-no-empty': null,
```

Comment:
I was initially surprised that we had empty comments but then pulled and saw where those are. Tbh I have a personal preference for `/* Long comment on multiple lines */` pattern but not sure how others feel about it and even if we decide to change the comments feels like a new PR

Reply:
Yeah I think we prefer `//` style comments because they are not output in the final CSS and we're largely documenting that Sass code here, where `/* */` is a real CSS comment and is output in the built file. Not super precious about comments though hence disabling the rule to let us do what makes sense.